src/modm/driver/display/st7586s.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021, Tomasz Wasilczyk
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+#define MODM_ST7586S_HPP
+
+#include <modm/ui/display/monochrome_graphic_display_horizontal.hpp>
+
+#include "st7586s_protocol.hpp"
+
+namespace modm
+{
+
+template <typename SPI, typename CS, typename RST, typename DC, int Width = 384, int Height = 160>
+// TODO: this controller has pixels packed by 3, not 8 per byte
```

Comment:
should be 4 not 8

Reply:
I mean MonochromeGraphicDisplay packs by 8. Is basically a todo to wait for your PR.